react/constants/Identifiers.ts

```diff
@@ -0,0 +1,4 @@
+export const PRODUCT_ID = 'ProductId'
+export const PRODUCT_SKU_ITEM_ID = 'ProductSkuItemId'
+export const PRODUCT_REFERENCE_ID = 'ProductReferenceId'
+export const PRODUCT_SKU_REFERENCE_ID = 'ProductSkuReferenceId'
```

Comment:
I'm not sure, but could this be changed to a `const enum`? 

```
export const enum IdentifierType {
  PRODUCT_ID = 'ProductId',
  PRODUCT_SKU_ITEM_ID = 'ProductSkuItemId',
  PRODUCT_REFERENCE_ID = 'ProductReferenceId',
  PRODUCT_SKU_REFERENCE_ID = 'ProductSkuReferenceId',
}
```

Reply:
@kaisermann fixed!